src/parser/parser.go

```diff
@@ -946,21 +947,15 @@ func (p *parser) parseStatement() ([]nodes.Node, error) {
 			case "autoescape":
 				return wrapInSlice(p.parseAutoescape)
 			default:
-				panic("unexpected statement keyword " + tokenValue)
```

Comment:
It's really an unexpected thing, it's an error in the parser (not in data feed to parser), i think it should panic

Reply:
youre right